spec/betterc.dd

```diff
@@ -0,0 +1,48 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generated light-weight,
+barebone, runtime-free binaries with are suitable for linking from C.
+
+---
+void main()
+{
+    import core.stdc.stdio : printf;
+    printf("Hello betterC");
+}
+---
+
+$(CONSOLE
+> dmd -betterC hello.d
+Hello betterC
+)
+
+$(H2 $(LNAME2 effects, Effects))
+
+    $(OL
+        $(LI Doesn't link DRuntime and standard library by default)
+	    $(LI Doesn't generate ModuleInfo entries)
+    )
+
+$(H2 $(LNAME2 consequences, Consequences))
+
+As no DRuntime is available, many D features won't work. For example:
+
+	$(OL
+	    $(LI No static module constructors or deconstructors)
+	    $(LI No Garbage Collection)
+	    $(LI No dynamic arrays and associative arrays)
+	    $(LI No Classes)
+	    $(LI No `synchronized`, monitors, mutex from $(MREF core, sync))
```

Comment:
Which reminds me that I need to move the core of the implementation of `core.sync.mutex` into a `struct` so it is usable in `betterC` for regular user-land programs that link libc.